Index: tests/phpunit/includes/utils.php
===================================================================
--- tests/phpunit/includes/utils.php	(revision 33328)
+++ tests/phpunit/includes/utils.php	(working copy)
@@ -375,7 +375,7 @@
 /**
  * Special class for exposing protected wpdb methods we need to access
  */
-class wpdb_exposed_methods_for_testing extends wpdb {
+class wpdb_exposed_methods_for_testing extends wpdb_drivers {
 	public function __construct() {
 		global $wpdb;
 		$this->dbh = $wpdb->dbh;
